x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -591,15 +592,18 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
                 border
                 subtitle={subTitle}
                 subtitle2={
-                  <>
-                    <EuiFlexGroup gutterSize="xs" alignItems="center" justifyContent="flexStart">
+                  <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexStart">
+                    {rule?.rule_source?.type === 'external' && rule.rule_source.is_customized && (
```

Comment:
There's a chance we will be doing these checks in multiple places. Let's create a domain-level function for checking if a given rule is customized, so the check here would be `isCustomizedPrebuiltRule(rule) && ...`

We can put this function under the `x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema` folder.

Reply:
Done!